releases/v20.2.14.md

```diff
@@ -0,0 +1,110 @@
+    <a href="https://binaries.cockroachdb.com/cockroach-v20.2.14.src.tgz"><button id="source" class="filter-button" data-scope="source" data-eventcategory="source-release-notes">Source</button></a>
+</div>
+
+<section class="filter-content" data-scope="windows">
+{% include windows_warning.md %}
+</section>
+
+### Docker image
+
+{% include copy-clipboard.html %}
+~~~shell
+$ docker pull cockroachdb/cockroach:v20.2.14
+~~~
+
+
+### Security updates
+
+- The `cert-principal-map` flag passed to [`cockroach` commands](../v20.2/cockroach-commands.html) now allows the certificate principal name to contain colons. [#67811][#67811]
+- The [node status retrieval endpoints over HTTP](../v20.2/monitoring-and-alerting.html) [`/_status/nodes`, `/_status/nodes/<N>` and the (DB Console](../v20.2/ui-overview.html) `/#/reports/nodes`) now require the [`admin` role](../v20.2/authorization.html) from the requesting user. This ensures that operational details such as network addresses and command-line flags do not leak to unprivileged users. [#67069][#67069]
+
+### Enterprise edition changes
+
+- [Incremental backups](../v20.2/take-full-and-incremental-backups.html) to a cloud storage location that already contains large existing backups now find their derived destination without listing as many remote files. [#67289][#67289]
+
+### SQL language changes
+
+- Added a new [`EXPLAIN` flag](../v20.2/explain.html), `MEMO`, to be used with `EXPLAIN (OPT)`. When the `MEMO` flag is passed, a representation of the optimizer memo will be printed along with the best plan. The `MEMO` flag can be used in combination with other flags such as `CATALOG` and `VERBOSE`. For example, `EXPLAIN (OPT, MEMO, VERBOSE)` will print the memo along with verbose output for the best plan. [#67778][#67778]
+
+### DB Console changes
+
+- The DB Console will now redirect the user to the originaly requested page after user login. [#67859][#67859]
+
+### Bug fixes
+
+- Fixed a statement buffer memory leak when using suspended portals. [#67370][#67370]
+- Correlated subqueries that couldn't be decorrelated and that have their own subqueries are now executed correctly when supported. Note that this is a very rare edge case. [#67571][#67571]
+- Fixed a very rare unexpected error from the [vectorized engine](../v20.2/vectorized-execution.html) ("index out of bounds") when evaluating the `CASE` operator. [#67785][#67785]
+- Catching up [Raft](../v20.2/architecture/replication-layer.html#raft) followers on the Raft log is now more efficient in the presence of many large Raft log entries. This helps avoid situations where Raft leaders struggle to retain leadership while catching up their followers. [#67126][#67126]
+- Fixed a rare bug in v20.2 that would happen when a lease change occured during a range merge. This bug allowed [`AS OF SYSTEM TIME`](..//v20.2/as-of-system-time.html) queries to serve inconsistent results. [#65823][#65823]
```

Comment:
Not sure on this one, but do we need "in v20.2"? I suppose my assumption is that all of these fixed bugs were in v20.2.